lib/vyasa_web/live/media_live/media_bridge.ex

```diff
@@ -150,6 +157,50 @@ defmodule VyasaWeb.MediaLive.MediaBridge do
     {:noreply, socket |> assign(should_show_vid: !flag)}
   end
 
+  @impl true
+  def handle_event(
+        "toggle_is_queue_visible",
+        _,
+        %{assigns: %{is_queue_visible: flag} = _assigns} = socket
+      ) do
+    {:noreply, socket |> assign(is_queue_visible: !flag)}
+  end
+
+  @impl true
+  def handle_event(
+        "hide_playback_queue",
+        _,
+        %{assigns: %{is_queue_visible: flag} = _assigns} = socket
+      ) do
+    IO.puts("HIDE PLAYBACK QUEUE")
+
+    case flag do
+      true -> {:noreply, socket |> assign(is_queue_visible: false)}
+      false -> {:noreply, socket}
+    end
+  end
+
+  @impl true
+  def handle_event(
```

Comment:
there's a state management bug here (that's why the double clicking is necessary)

easiest signal for this is to look at the vissual debug dump and realise the value for :is_queue_visible doesn't toggle correctly 

two actions to consdier:

1. clicking the button @ action bar
2. click-out from the playback queue



i haven't investigated the state management bug actually, so not sure as to the cause of it yet


this is an annoyance, but it can be tolerated, we can wontfix it for the meantime and this is low priority.